docs-source/cloudbank/content/saga/business-logic.md

```diff
@@ -12,141 +12,141 @@ The deposit service will be responsible for depositing funds into accounts. It w
 
   This method should write a journal entry for the deposit, but should not update the account balance.  Here is the code for this method:
 
-    ```java
-    
-    /**
-     * Write journal entry re deposit amount.
-     * Do not increase actual bank account amount
-     */
-    @PostMapping
-    @LRA(value = LRA.Type.MANDATORY, end = false)
-    public ResponseEntity<String> deposit(@RequestHeader(LRA_HTTP_CONTEXT_HEADER) String lraId,
-            @RequestParam("accountId") long accountId,
-            @RequestParam("amount") long depositAmount) {
-        log.info("...deposit " + depositAmount + " in account:" + accountId 
-            + " (lraId:" + lraId + ") finished (in pending state)");
-        Account account = AccountTransferDAO.instance().getAccountForAccountId(accountId);
-        if (account == null) {
-          log.info("deposit failed: account does not exist");
-          AccountTransferDAO.instance().saveJournal(
-            new Journal(
-                DEPOSIT, 
-                accountId, 
-                0, 
-                lraId,
-                AccountTransferDAO.getStatusString(ParticipantStatus.Active)
-            )
-          );
-          return ResponseEntity.ok("deposit failed: account does not exist");
-        }
-        AccountTransferDAO.instance().saveJournal(
-          new Journal(
-              DEPOSIT, 
-              accountId, 
-              depositAmount, 
-              lraId,
-              AccountTransferDAO.getStatusString(ParticipantStatus.Active)
-          )
-        );
-        return ResponseEntity.ok("deposit succeeded");
+```java
```

Comment:
i think the indenting is wrong here - run it locally (`hugo serve`) and check. applies to several other places too. it needs to be "inside" the numbered list item so that it does not reset the numbering and mess up the left margin

Reply:
I'll retry this with the hugo server